sdk-http-apache-async/src/main/java/io/sphere/sdk/http/ApacheHttpClientAdapterImpl.java

```diff
@@ -97,7 +99,10 @@ private HttpUriRequest toApacheRequest(final HttpRequest httpRequest) throws Uns
             final HttpRequestBody body = httpRequest.getBody();
             final HttpEntity httpEntity;
             if (body instanceof StringHttpRequestBody) {
-                httpEntity = stringEntityOf(((StringHttpRequestBody) body).getString());
+                StringEntity stringEntity = new StringEntity(((StringHttpRequestBody) body).getString(),"UTF-8");
```

Comment:
didn't know about the charset thing 👍 
but its better because now the code is even safer